inductiva/utils/files.py

```diff
@@ -108,80 +107,41 @@ def get_alphanum_key(file):
     return files
 
 
-def _unzip(zip_path: pathlib.Path, dest_path: pathlib.Path):
-    """Unzip a zip archive.
+def _unzip(zip_path: pathlib.Path):
+    """Unzip a zip archive and remove the .zip."""
 
-    If the ZIP contains one file, dest_path will be the path of that single
-    file. If the ZIP contains multiple files, dest_path will be the path
-    of the directory containing the files.
-    """
     with zipfile.ZipFile(zip_path, "r") as zip_ref:
-        files = zip_ref.namelist()
-        if len(files) == 1:
-            # Unzip the file to the directory of dest_path
-            unzipped_file_path = pathlib.Path(
-                zip_ref.extract(files[0], dest_path.parent))
-            pathlib.Path(unzipped_file_path).replace(dest_path)
-        else:
-            dest_path.mkdir(parents=True, exist_ok=True)
-            zip_ref.extractall(dest_path.parent)
-
-    # Rename the file to dest_path
+        zip_ref.extractall()
+
     zip_path.unlink()
 
 
-def download_from_url(url: str, save_dir: Optional[str] = None) -> str:
+def download_from_url(url: str, unzip: bool = False) -> str:
```

Comment:
Just curious, Is there any reason why the users would not want to unzip it right away?

Reply:
This can be used for several things, and the unzip tells they users they are actually uncompressing.